common/src/main/java/com/revenuecat/purchases/common/offlineentitlements/OfflineEntitlementsManager.kt

```diff
@@ -92,6 +95,8 @@ class OfflineEntitlementsManager(
             )
         }
     }
+
+    private fun isOfflineEntitlementsEnabled() = appConfig.finishTransactions
```

Comment:
We don't store observer mode directly, but we store the `finishTransactions` flag instead.

Reply:
Yeah... It's basically the same, but I guess we can have both, so it's easier to understand 👍